packages/browser-utils/tsconfig.test.json

```
@@ -1,11 +1,11 @@
 {
   "extends": "./tsconfig.json",
 
-  "include": ["test/**/*"],
+  "include": ["test/**/*", "vite.config.ts"],
 
   "compilerOptions": {
     // should include all types from `./tsconfig.json` plus types for all test frameworks used
-    "types": ["node", "jest"]
+    "types": ["node", "jest", "vitest"]
```

```suggestion
    "types": ["node", "vitest"]
```

can/should we remove jest here?